src/isomorphic/classic/element/ReactDebugCurrentFrame.js

```diff
@@ -38,7 +38,9 @@ if (__DEV__) {
       if (typeof current === 'number') {
         // DebugID from Stack.
         const debugID = current;
-        stack = getStackAddendumByID(debugID);
+        if (getStackAddendumByID) {
```

Comment:
This check seems unnecessary, other than for flow, since we're within a `__DEV__` block. I liked Dan's suggestion below- to use a different type for dev/prod so we can avoid all of the unnecessary not-defined checks below.